lib/generators/react_on_rails/templates/redux/base/client/app/lib/middlewares/loggerMiddleware.js

```diff
@@ -12,7 +12,7 @@ export default function logger({ getState }) {
 
     // We can't console.log immutable objects out-of-the-box.
     const immutableState = getState();
-    const readableState = _.reduce(immutableState, (result, immutable, key) => {
+    const readableState = _.reduce(immutableState, (object, immutable, key) => {
       result[key] = immutable.toJS();
```

Comment:
@justin808 This is something I actually wanted to talk to @alexfedoseev about on Skype or something. It's a really weird situation going on with variable shadowing and scope that's actually interesting in just understanding how JavaScript handles this scenario. Even when what is now `object` was also called `result`, despite my intuition, `result[key]` was referring to the object *outside* the function and I believe this is the intended behavior. In other words, in both versions, what is now `object` was never being used.